src/ucp/api/ucp.h

```diff
@@ -277,6 +277,11 @@ enum ucp_ep_params_flags_field {
                                                            send to a particular
                                                            remote endpoint, for
                                                            example stream */
+    UCP_EP_PARAMS_FLAGS_NO_CONN_MATCH  = UCS_BIT(2)   /**< Avoid adding the endpoint
```

Comment:
conn_match is an internal structure, so the flag and its documentation should be according to their implications on API level. something like:
```
/* Do not attempt to match this endpoint to a remote endpoint created between the same
   pair of workers.
   This connection mode makes sure this endpoint is not affected if any remote endpoint is 
   closed, but can consume more transport resources, and cannot be used for @ref ucp_stream_send 
   and @ref ucp_stream_recv operations */
UCP_EP_PARAMS_FLAGS_NO_PEER_MATCH
```